customTests/azure_cpu_stream.nhc

```diff
@@ -61,13 +26,25 @@ function get_core_conf(){
 }
 
 function install_check(){
-    EXE_DIR=$1
+    HB_HX_SKUS="standard_hb176rs_v4|standard_hb176-144rs_v4|standard_hb176-96rs_v4|standard_hb176-48rs_v4|standard_hb176-24rs_v4|standard_hx176rs|standard_hx176-144rs|standard_hx176-96rs|standard_hx176-48rs|standard_hx176-24rs"
+    SKU=$( curl -H Metadata:true --max-time 10 -s "http://169.254.169.254/metadata/instance/compute/vmSize?api-version=2021-01-01&format=text")
+    SKU=$(echo "$SKU" | tr '[:upper:]' '[:lower:]')
+    if [[ "$HB_HX_SKUS" =~ "$SKU"  ]]; then
+        stream=stream4.exe
+    elif [[ "$SKU" == *"hb120"*"_v3"* ]]; then
+        stream=stream3.exe
+    elif [[ "$SKU" == *"hb120rs_v2"* ]]; then
```

Comment:
This is the only hbv2 sku we care about?

Reply:
Yea, currently we don't support the other HBv2 sizes.